pulseapi/entries/views.py

```diff
@@ -241,6 +241,8 @@ class EntriesListView(ListCreateAPIView):
     search_fields = (
         'title',
         'description',
+        'get_involved',
```

Comment:
please also update https://github.com/arush0311/network-pulse-api/blob/79bbbd7b9f80543dbdebfa97738f88ab8eaf17e5/pulseapi/entries/views.py#L220 so that the fact that these are searched is documented when people open the DRF page on `/entries`